File: 0268-missing-number/0268-missing-number.cs
public class Solution {
    public int MissingNumber(int[] nums) {
     int n=nums.Length;
     int sum=0;
     int totalSum=n*(n+1)/2;
     for(int i=0;i<nums.Length;i++){
        sum+=nums[i];
     } 
     return totalSum-sum;  
    }
}

File: 0268-missing-number/0268-missing-number.ts
function missingNumber(nums: number[]): number {
    let n=nums.length;
    let arraySum=0;
    let NnumTotal=n*(n+1)/2;
    for(let i=0;i<nums.length;i++){
        arraySum+=nums[i];
    }
    return NnumTotal-arraySum;
};